OpenShift/helm/nexus-repository-manager/templates/service.yaml

```diff
@@ -1,40 +1,11 @@
-{{- if .Values.service.enabled -}}
 apiVersion: v1
 kind: Service
 metadata:
-{{- if .Values.service.name }}
-  name: {{ .Values.service.name }}
-{{- else }}
-  name: {{ template "nexus.name" . }}-service
-{{- end }}
-  labels:
-{{ include "nexus.labels" . | indent 4 }}
-{{- if .Values.service.labels }}
-{{ toYaml .Values.service.labels | indent 4 }}
-{{- end }}
-{{- if .Values.service.annotations }}
-  annotations:
-{{ toYaml .Values.service.annotations | indent 4 }}
-{{- end }}
+  name: {{ template "nexus.fullname" . }}-service
 spec:
-  ports:
-  {{- if .Values.service.portName }}
-  - name: {{ .Values.service.portName }}
-    port: {{ .Values.service.port }}
-    targetPort: {{ .Values.service.targetPort }}
-  {{- end }}
-  {{- with .Values.service.ports  }}
-{{ toYaml . | indent 2 }}
-  {{- end }}
   selector:
     app: {{ template "nexus.name" . }}
-    release: {{ .Release.Name }}
-  type: {{ .Values.service.serviceType }}
-  {{ if .Values.service.loadBalancerSourceRanges }}
-  loadBalancerSourceRanges:
-    {{- range .Values.service.loadBalancerSourceRanges }}
-    - {{ . }}
-    {{- end }}
-  {{ end }}
-{{- end}}
-
+  ports:
+    - name: application
+      port: {{ .Values.nexus.nexusPort }}
+  type: ClusterIP
```

Comment:
this whole file has been stripped down to look like what IQ unconditionally does to create a service, so this eliminates the need to `kubectl expose` anymore.